core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/YouTubePlayerView.kt

```diff
@@ -98,6 +101,29 @@ class YouTubePlayerView(context: Context, attrs: AttributeSet? = null, defStyleA
         })
     }
 
+    /*Ontouch event override function*/
+    override fun dispatchTouchEvent(event: MotionEvent): Boolean {
```

Comment:
Can't we just add an `OnTouchListener` and trigger the callback from there?

Reply:
Actually onTouch listener gets called in from this dispatchTouchEvent, I can send call back to OnTouchListener by checking the action Up OR Down by calling below function, 
    override fun performClick(): Boolean {
        return super.performClick()
    }
it returns the onTouchListener event in view, but in our case some how it is not returning to setOnTouchListener call back in our view that's why I have created my own callback.